mobile/src/main/java/org/openhab/habdroid/core/connection/ConnectionFactory.kt

```diff
@@ -372,6 +373,8 @@ class ConnectionFactory internal constructor(
                     active?.remote?.toCloudConnection()
                 }
                 updateActiveCloud(CloudConnectionResult(result, null))
+            } catch (e: CancellationException) {
```

Comment:
From the logs I posted, it seems to be JobCancellationException that is thrown, not CancellationException.

Reply:
The `internal` `JobCancellationException` is a child class of `CancellationException`, so I had to use that.